Program.cs

```diff
@@ -200,6 +247,104 @@ static void Main(string[] args)
             }
         }
 
+        private static int GetMetaDataEpoch(
+            string outputDirectory,
+            string epochType)
+        {
+            try
+            {
+                string previousMetadata = Directory.GetFiles(outputDirectory)
+                    .Where(x => Path.GetExtension(x) == ".json")
+                    .OrderByDescending(x => File.GetLastWriteTime(x))
+                    .First();
+                using (StreamReader reader = new StreamReader(previousMetadata))
+                {
+                    var jsonString = reader.ReadToEnd();
+                    var jsonObject = JObject.Parse(jsonString); 
+                    return (int)jsonObject[epochType];            
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
```

Comment:
IMO it should print into stderr, not stdout.
```suggestion
                Console.Error.WriteLine(e.Message);
```